doc/content/microreactors/gcmr/GCMR_Multiphysics_models.md

```diff
@@ -0,0 +1,174 @@
+## Mesh files
+
+The 3-D mesh for the GC-MR assembly displayed in [Fig_2] was generated by MOOSE intrinsic meshing tools available in the [`Reactor module`](https://mooseframework.inl.gov/modules/reactor/index.html). This MOOSE-based tool allows a direct construction of assembly or reactor core mesh from simple cartesian and hexagonal unit geometries. The Reactor Module consists of a series of mesh generators to enable meshing of reactor cores. Using MOOSE mesh generators for constructing meshes for reactor geometries entails following the intrinsic geometric hierarchy where the unit cartesian or hexagonal structures (e.g., unit pin-cell structures of control rod, fuel pin) can be assembled into fuel assemblies. The objects in Reactor Module provides direct control of mesh/element size through the input parameters. Adjusting the mesh density from coarse to fine takes little time, streamlining sensitivity analyses on mesh refinement. The consistency between the geometrical representations in the mesh file and the cross-section generation code was verified by directly comparing the volumes of the different blocks/universes. In this example, only the fine mesh is needed for the Griffin input, which is given in `Griffin_mesh.e` , and Griffin generates the coarse mesh required for the CMFD acceleration as such:
+
+!listing
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = '../MESH/ Griffin_mesh.e'
+  []
+  [fmg_id]
+    type = SubdomainExtraElementIDGenerator
+    input = fmg
+    subdomains = '10 100 102 103 200 201 300 400 401 500 600 8000 8001'
+    extra_element_id_names = 'material_id equivalence_id'
+    extra_element_ids = '803 802 810 806 807 807 804 801 801 808 809 805 805;
+                         803 802 810 806 807 807 804 801 801 808 809 805 805'
+  []
+  [coarse_mesh]
+    type = GeneratedMeshGenerator
+    dim= 3
+    nx = 10
+    ny = 10
+    nz = 20
+    xmin = -0.1
+    xmax =  0.1
+    ymin = -0.1
+    ymax =  0.1
+    zmin =  0.
+    zmax =  2.
+  []
+  [assign_coarse_id]
+    type = CoarseMeshExtraElementIDGenerator
+    input = fmg_id
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+[]
```

Comment:
use the blocks from the input file directly

!listing path to input block=Mesh

Reply:
same for all the other listings

Reply:
Thank you for the comment. I applied it.